doc/discretization.rst

```diff
@@ -132,96 +146,413 @@ $$
+   $\b{H}_s$, should be equal and opposite.
 
-In DG, numerical fluxes are not only responsible for handling the flow of information
-between adjacent cells, but they also enforce information flow at the boundaries.
+Domain boundary treatments
+==========================
+
+What happens when $\partial E \cap \partial\Omega \neq \emptyset$?
 
-We denote the *boundary fluxes* as $\b{h}^*_e(\b{Q}_{bc})$,
-$\b{h}^*_v(\b{Q}_{bc}$, $\b{\Sigma}_{bc})$, and
-$\b{H}^*_s(\b{Q}_{bc})$, where $\b{Q}_{bc}$, $\b{\Sigma}_{bc}$ denote
-boundary conditions imposed on the state, and the gradient of the state respectively.
+In DG, fluxes are responsible for handling the flow of information
+between adjacent cells, and for transferring the boundary conditions
+into the domain from the domain boundary.  In this sense, all of the
+boundary conditions are *weak*, in that they are weakly enforced
+through the flux, as opposed to directly setting the solution of the
+boundary elements (a *strong* enforcement).
+
+Boundary treatments in *MIRGE-Com* follow the prescriptions of the
+the so-called BR1 method descibed by [Bassi_1997]_, and the boundary
+treatment strategies outlined by [Mengaldo_2014]_.
+
+The relevant quantities for the boundary treatments are as follows:
+
+.. math::
+   
+  \b{Q}^{\pm} &\equiv \text{conserved quantities on the exterior/interior of the boundary face} \\
+  \b{Q}_{bc} &\equiv \text{boundary condition for the fluid conserved quantities} \\
+  \b{\Sigma}^{\pm} &\equiv \text{gradient of conserved quantities on ext/int of boundary face} \\
+  \b{\Sigma}_{bc} &\equiv \text{boundary condition for grad of soln } \\
+  \b{v}^{\pm} &\equiv \text{Flow velocity on the ext/int of boundary face} \\
+  \b{h}^*_e &\equiv \text{boundary flux for the divergence of inviscid flux} \\
+  \b{h}^*_v &\equiv \text{boundary flux for divergence of viscous flux} \\
+  \b{H}_s^* &\equiv \text{boundary flux for the gradient of the conserved quantities} \\
+  \hat{\b{n}} &\equiv \text{outward pointing normal for the boundary face}
 
 For all $\partial E \cap \partial\Omega$ the $+$ side is on the domain boundary. 
-Boundary conditions are set by prescribing one or more components of the solution
-or its gradient, $\b{Q}^+ = \b{Q}_{bc}$, and $\b{\Sigma}^+ = \b{\Sigma}_{bc}$, respectively,
-or by prescribing one or more components of the boundary fluxes $\b{h}^*_e$, $\b{h}^*_v$,
-and $\b{H}^*_s$.  Descriptions of particular boundary treatments follow in the next few
-sections.
+Boundary conditions ($\b{Q}_{bc}, \b{\Sigma}_{bc}$) are set by prescribing one or more
+components of the solution or its gradient on the (+) side of the boundary,
+($\b{Q}^+, \b{\Sigma}^+$), respectively, or by prescribing one or more components of the
+boundary fluxes $\b{h}^*_e$, $\b{h}^*_v$, and $\b{H}^*_s$.  Descriptions of particular
+boundary treatments follow in the next few sections.
 
+.. image:: figures/ElementBoundary.png
```

Comment:
Track down the source file?

Reply:
https://sphinxcontrib-tikz.readthedocs.io/en/latest/

Reply:
> Track down the source file?

I have it, if it is useful.  PPTX

Reply:
#667